scripts/landing.js

```diff
@@ -9,4 +10,16 @@ var animatePoints = function() {
   for (var i = 0; i < points.length; i++) {
     revealPoint(i);
   }
-};
\ No newline at end of file
+};
+window.onload = function() {
+  if (window.innerHeight > 950) {
+    animatePoints(pointsArray);
+  }
+  var sellingPoints = document.getElementsByClassName('selling-points')[0];
+  var scrollDistance = sellingPoints.getBoundingClientRect().top - window.innerHeight + 200;
```

Comment:
top value of .selling-Points  minus window.innerHeight plus 200

//getBoundingClientRect() returns left, top, right, bottom, x, y, width, height